src/load_stats.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+import csv
+import gzip
+import os
+import sys
+
+from config import *
+
+
+def load_stats(output_file):
+    #   ____                   __     ___
+    #  / ___| _   _ _ __ ___   \ \   / (_) _____      _____
+    #  \___ \| | | | '_ ` _ \   \ \ / /| |/ _ \ \ /\ / / __|
+    #   ___) | |_| | | | | | |   \ V / | |  __/\ V  V /\__ \
+    #  |____/ \__,_|_| |_| |_|    \_/  |_|\___| \_/\_/ |___/
+    #
+
+    stats = {site: dict() for site in STATS_SITES}
+
+    for i, dump_file in enumerate(STATS_DUMP_FILES):
+        path = os.path.join(STATS_DUMP_DIR, dump_file)
+        print('({}/{}) {}'.format(i + 1, len(STATS_DUMP_FILES), path))
+
+        for i, line in enumerate(gzip.open(path)):
+            try:
+                site, page, views, responses = line.decode().strip().split()
+            except Exception as e:
+                print('Error at line', i, ':', e, file=sys.stderr)
+
+            page = page.replace('_', ' ')
```

Comment:
in case of an error, `page` may undefined here

Reply:
I fixed my linter's configuration